changelog.md

```diff
@@ -1,5 +1,95 @@
+- Fix #14800: Incorrect register processing in GetCustomStationRelocation (#14801)
+- Fix #14755: Remove clicked type selection when not visible (#14796)
+- Fix #14794: Default railtype missing for some NewGRF rail vehicles (#14795)
+- Fix: Incorrect parsing of var 6x parameter in NewGRF debug window (#14789)
+- Fix: Two lighthouse spawn issues (#14785)
+- Fix #14777: Authorized_key: Correctly target key type for add/remove (#14778)
+- Fix: Incorrect spacing for badges in dropdown lists (#14768)
+- Fix: Unconfigured badge classes should be visible in column 0 by default (#14766)
+- Fix #14763: Crash if NewGRF currency separator is not valid (#14764)
+- Fix #14701: Company colour remap for sprites in badge filter dropdowns (#14732)
+- Fix: Do not pre-fill industry production history for unused production slots (#14730)
+- Fix: Don't set set town index for depot tiles (#14729)
+- Fix #14723: Set force proceed to TFP_SIGNAL if next tile has signals (#14724)
+- Fix: Miscalculated cargo penalty for poor station rating (#14712)
+- Fix: Crash when user enters a blank line in the console (#14711)
+- Fix: Dump_info should not reverse non-ASCII label (#14697)
+- Fix: Incorrect parameter order for CmdSetCompanyManagerFace (#14695)
+- Fix: Bootstrap ignored default OpenTTD truetype fonts (#14684)
+- Fix: League Table layout broken with RTL languages (#14667)
+- Fix #14549: Changing interface scale could underflow map zoom (#14655)
+- Fix: Incorrect row height in network server list (#14653)
+- Fix: Doubled beep sounds when clicking toolbar buttons (#14642)
+- Fix: Wrong button type for town menu in scenario editor toolbar (#14641)
+- Fix #14631: Waypoint customs spec not allocated properly on initial construction (#14633)
+- Fix: Variant cycle detection in FinaliseEngineArray (#14629)
+- Fix: Game crash due to invalid vehicle type information (#14628)
+- Fix #14620: Use full file path when deleting files (#14623)
+- Fix: [Script] Return rail types as list instead of bitmask (#14617)
+- Fix #14604: Clearing tiles to build objects did not update town ratings (#14616)
+- Fix: Bridge height check for waypoints didn't include axis in layout (#14609)
+- Fix #14607: Bridge-over-station discrepancy depending on build order (#14608)
+- Fix: Don't add spacing in rail/road type dropdowns if no badges are present (#14598)
+- Fix: [AI] Incorrect infrastructure cost for road/tram tiles (#14596)
+- Fix #14588: Show error when unable to clone partly-cleared crashed train (#14591)
+- Fix #14586: Empty station tile layouts incorrectly substituted with default layouts (#14587)
+- Fix #14584: Crash due to drawing non-existent orders of new vehicle (#14585)
+- Fix #14572: Incorrect playlist entry was removed if there are duplicates (#14583)
+- Fix: Wrong row may be selected in music playlists (#14581)
+- Fix #14569: Ensure music playlist window is large enough (#14570)
+- Fix #14278: [Script] Don't set members inside operator new() (#14568)
```

Comment:
`Fix #14278: [Script] Memory allocation limit did not work and could result in a crash. (#14568)`